README.md

```diff
@@ -2,7 +2,9 @@
 
 A barebones Node.js app using [Express](https://expressjs.com/).
 
-This application supports the [Getting Started on Heroku with Node.js](https://devcenter.heroku.com/articles/getting-started-with-nodejs) article - check it out.
+This application supports the [Getting Started on Heroku with Node.js](https://devcenter.heroku.com/articles/getting-started-with-nodejs) 
+and [Getting Started on Heroku Fir with Node.js](https://devcenter.heroku.com/articles/getting-started-with-nodejs-fir) 
+articles - check it out.
 
```

Comment:
How about this?

This application supports the Getting Started guides for both the [Cedar and Fir generations](https://devcenter.heroku.com/articles/generations) of the Heroku platform. You can check them out here:

* [Getting Started on Heroku with Node.js](https://devcenter.heroku.com/articles/getting-started-with-nodejs)
* [Getting Started on Heroku Fir with Node.js](https://devcenter.heroku.com/articles/getting-started-with-nodejs-fir)

Reply:
Not that you have to, but you can make a multi-line suggestion by highlighting the line numbers and clicking the little blue "+" to open a comment

<img width="945" alt="image" src="https://github.com/user-attachments/assets/46c71215-b2c4-478c-8270-07679ceb6401">

Then you can click "suggest a  change" 

<img width="994" alt="image" src="https://github.com/user-attachments/assets/60e61c02-612e-4440-86ed-ead418cdacc1">

It gives you a little suggestion box with the original

<img width="681" alt="image" src="https://github.com/user-attachments/assets/5f05f6c9-9975-4cf9-9200-caa4c25cfc05">

If you make changes to it and leave a comment, then the original person who wrote the PR can apply it directly if they like it. I turned your comment into a suggested change here https://github.com/heroku/node-js-getting-started/pull/348/files/22e5734e1c7fa1ade2b9570d93fbb58cb3de1a47#r1870321855. I like the bullet points, I think it makes it a little more clear than having it inline.